toolkit/docs/building/building.md

```diff
@@ -479,6 +482,8 @@ If that is not desired all remote sources can be disabled by clearing the follow
 > - `mariner-ms-non-oss.repo` and `mariner-ms-non-oss-preview.repo` - Azure Linux repository containing Microsoft non open Source RPMs with sources not viewable to the public. The preview version serves the same purpose as the official preview repo.
 >
 
+`DAILY_BUILD_ID` and `DAILY_BUILD_REPO` will automiatically augment the `REPO_LIST`.
```

Comment:
typo automiatically

Reply:
Hah, last minute doc updates are a mistake, especially with the spellcheck off.